packages/fxa-content-server/tests/functional/settings_v2/two_step_auth.js

```diff
@@ -0,0 +1,157 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { describe, it, before } = intern.getPlugin('interface.bdd');
+const selectors = require('../lib/selectors');
+const FunctionalHelpers = require('../lib/helpers');
+const FunctionalSettingsHelpers = require('./lib/helpers');
+const { navigateToSettingsV2 } = FunctionalSettingsHelpers;
+const { generateTotpCode } = FunctionalHelpers;
+let secret;
+let securityCode;
```

Comment:
These two variables do not need to be in this scope.  You can declare and assign them as `const`s where they are first used.

Reply:
Made the change. Thanks for the review! :)